manifests/h/h3poteto/whalebird-desktop/v5.0.4/h3poteto.whalebird-desktop.yaml

```
@@ -0,0 +1,8 @@
+# Created with WinGet Releaser v2 using Komac v1.5.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: h3poteto.whalebird-desktop
+PackageVersion: v5.0.4

```

```suggestion
PackageVersion: 5.0.4
```
This must match Control Panel.